app.js

```diff
@@ -4,13 +4,12 @@ var path = require('path');
 var cookieParser = require('cookie-parser');
```

Comment:
Not required for Passport since Express 1.5, according to Passport documentation. Will check and let you know now.

Reply:
Ok it has been removed